src/vsr/replica.zig

```diff
@@ -138,13 +138,16 @@ pub fn ReplicaType(
         /// The current view, initially 0:
         view: u32,
 
-        /// The latest view, in which the replica's status was normal.
-        view_normal: u32,
+        /// The latest view where
+        /// - the replica was a primary and acquired a DVC quorum, or
+        /// - the replica was a backup and received a SV message.
+        log_view: u32,
```

Comment:
Can we add why we care about this. I think this is correct:

```suggestion
        /// ie the last view for which we can guarantee that this replica was not missing any committed ops.
        log_view: u32,
```

Reply:
I guess that's not quite correct - if it was the primary it might set log_view before completing repair?

Reply:
Right, it should be more along the lines of:

> ie the latest view in which this replica changed its head message

The primary of a new view updates its log_view before completing repair, and followers joining a view (via `start_view`) update their `log_view` immediately, and then repair asynchronously.